disnake/ui/view.py

```diff
@@ -356,7 +355,7 @@ async def _scheduled_task(self, item: Item, interaction: MessageInteraction) ->
 
             allow = await self.interaction_check(interaction)
             if not allow:
-                return
+                return None
```

Comment:
This method also doesn't return any value, so plain `return` should be used instead